app/static/src/app/utils.ts

```diff
@@ -31,6 +31,14 @@ export interface ProcessFitDataResult {
     timeVariableCandidates: string[] | null,
     error?: Error
 }
+
+export function getFormedError(errorResponse: OdinModelResponseError): Error {
+    const line = errorResponse.line.join();
+    const detail = line ? `Error on line ${line}: ${errorResponse.message}` : errorResponse.message;
+    const error = "OTHER_ERROR";
```

Comment:
I don't think we need to have `OTHER_ERROR` here - that's kind of just a hangover from HINT error codes. We could just have something sensible and human readable like "Code error" - but actually, that might not be all that useful anyway. I think it would be fine to not try to force the code error into a standard error for displaying in ErrorInfo, and just have a custom display for the code error. In that case you wouldn't really need to keep it separately in the state, and could just format it for display in the component. (Though, apologies, I think I suggested doing it this way in the first place!) Up to you.